lib/heroku-url.js

```diff
@@ -1,3 +1,4 @@
 module.exports = function() {
-  return "https://arcane-depths-57821.herokuapp.com/";
+  //return "https://arcane-depths-57821.herokuapp.com/";
+  return "http://localhost:3000/";
```

Comment:
Good job on keeping this separate. We can make the switch back to the heroku server when we need, but while we are testing through I like just losing local host to not disrupt our production db